src/System.Data.SqlClient/src/Interop/LocaleMapperApiHelper.cs

```diff
@@ -0,0 +1,482 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
I like changing LocaleInterop to use these helpers, since it makes it clear what the OS differences are. LocaleInterop is just our caller and the helpers define the specific functionality we need, so you can just look at them when working on an OS-specific feature. It's just like separating out protocol functionality with our SNIHandles.